org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/thingTranslator.js

```diff
@@ -1,16 +1,17 @@
 /* jshint esversion: 6 */
 /* jshint node: true */
 'use strict';
+var OpenT2TErrorClass = require('opent2t').OpenT2TError;

```

Comment:
so followed pattern in hubcontroller.js but will fix tx

---
In reply to: [101369065](https://github.com/openT2T/translators/pull/158#discussion_r101369065) [](ancestors = 101369065,101368120)